lib/rules/no-octal-escape.js

```diff
@@ -25,7 +25,7 @@ module.exports = {
         return {
 
             Literal(node) {
-                if (typeof node.value !== "string") {
+                if (typeof node.value !== "string" || node.raw === undefined) {
```

Comment:
We don't like using the `undefined` identifier (as shown by the lint errors in Travis/AppVeyor). Instead, we prefer `typeof node.raw === "undefined"`.